src/components/PullToRefresh/index.tsx

```diff
@@ -0,0 +1,52 @@
+import Router from 'next/router';
+import PR from 'pulltorefreshjs';
+import { useEffect } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { RefreshIcon } from '@heroicons/react/outline';
+
+const PullToRefresh: React.FC = () => {
+  useEffect(() => {
+    PR.init({
+      mainElement: '#pull-to-refresh',
+      onRefresh() {
+        Router.reload();
+      },
+      iconArrow: ReactDOMServer.renderToString(
+        <div className="z-50 flex items-center justify-center">
+          <RefreshIcon className="z-50 h-9 w-9 rounded-full border-4 border-indigo-600 bg-indigo-600 text-white shadow-md" />
+        </div>
+      ),
+      iconRefreshing: ReactDOMServer.renderToString(
+        <div className="z-50 flex items-center justify-center">
+          <RefreshIcon
+            className="z-50 h-9 w-9 animate-spin rounded-full border-4 border-indigo-600 bg-indigo-600 text-white shadow-md"
+            style={{ animationDirection: 'reverse' }}
+          />
+        </div>
+      ),
+      instructionsPullToRefresh: ReactDOMServer.renderToString(
+        <div>
+          <p className="hidden"></p>
+        </div>
+      ),
+      instructionsReleaseToRefresh: ReactDOMServer.renderToString(
+        <div>
+          <p className="hidden"></p>
+        </div>
+      ),
+      instructionsRefreshing: ReactDOMServer.renderToString(
+        <div>
+          <p className="hidden"></p>
+        </div>
+      ),
```

Comment:
```suggestion
      iconArrow: ReactDOMServer.renderToString(
        <RefreshIcon className="z-50 m-auto h-9 w-9 rounded-full border-4 border-indigo-800 bg-indigo-800 text-indigo-100 shadow-md" />
      ),
      iconRefreshing: ReactDOMServer.renderToString(
        <RefreshIcon
          className="z-50 m-auto h-9 w-9 animate-spin rounded-full border-4 border-indigo-800 bg-indigo-800 text-indigo-100 shadow-md"
          style={{ animationDirection: 'reverse' }}
        />
      ),
      instructionsPullToRefresh: ReactDOMServer.renderToString(<div />),
      instructionsReleaseToRefresh: ReactDOMServer.renderToString(<div />),
      instructionsRefreshing: ReactDOMServer.renderToString(<div />),
```
Can clean this up a little bit 😊 And I'm still not sure about the colors, but maybe darker (as you suggested) is better? 🤔 How do you think this looks?

Reply:
Pushed changes with new icon. Updated main post video to reflect.

Reply:
Oh I think that looks better! Maybe a slightly brighter arrow color though? I don't think there's enough contrast with the gray 🤔

Reply:
Done.